tasks/simplifications.json

```diff
@@ -183,7 +183,7 @@
 		"maxmemory": "8192",
 		"skipfailures": "no"
 	},
-	"ca.statcan.geo.importgeography_pr_0": {
```

Comment:
Why is this change in this PR? Shouldn't be [here](https://github.com/CartoDB/bigmetadata/issues/383)?